NCTRC/frontend/screens/AdminPasswordScreen.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+import { SafeAreaView, StyleSheet, Text, View, Button, Image, TextInput, TouchableOpacity } from 'react-native';
+import { StackNavigationProp } from '@react-navigation/stack';
+import { AuthStackParamList, AppScreens } from '../Index';
+import { styles } from './Styles';
+
+type AdminPasswordScreenNavigationProps = StackNavigationProp<AuthStackParamList, AppScreens.AdminPass>;
+interface AdminPasswordScreenProps {
+    navigation: AdminPasswordScreenNavigationProps;
+}
+const AdminPassScreen: React.FunctionComponent<AdminPasswordScreenProps> = (props) => {
+    const { navigation } = props;
+  
+    return (
+        <SafeAreaView style={styles.container}>
+        <TouchableOpacity style={styles.backButton} onPress={() => navigation.pop()}>
+                 <Image source={require('./../assets/backbutton.png')} style={{ width: 75, height: 75 }}></Image>
```

Comment:
I think we should probably import the image instead of requiring it, thoughts?

Reply:
I think you would import a new component with the image and would still have to require in that component.